apps/frontend/src/renderer/components/project-wizard/GitLabStep.tsx

```
@@ -0,0 +1,712 @@
+                    {isLoadingProjects ? (
+                      <div className="flex justify-center py-8">
+                        <Loader2 className="h-8 w-8 animate-spin text-primary" />
+                      </div>
+                    ) : projects.length > 0 ? (
+                      <div className="max-h-80 overflow-y-auto space-y-2 mb-6">
+                        {projects.map((proj) => (
+                          <button
+                            key={proj.pathWithNamespace}
+                            onClick={() => handleSelectProject(proj.pathWithNamespace)}
+                            className="w-full flex items-center justify-between p-4 rounded-lg border border-border hover:bg-accent transition-colors text-left"
+                          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)